src/Symfony/Component/HttpClient/HttpClientTrait.php

```diff
@@ -180,7 +180,7 @@ private static function mergeDefaultOptions(array $options, array $defaultOption
             }
 
             if ('auth_ntlm' === $name) {
-                throw new InvalidArgumentException(sprintf('Option "%s" is not supported by %s, try using CurlHttpClient instead.', __CLASS__));
+                throw new InvalidArgumentException(sprintf('Option "%s" is not supported by %s, try using CurlHttpClient instead.', $name, __CLASS__));
```

Comment:
let's inline the value instead? $name is auth_ntlm

Reply:
Sure, just updated it.